chrome/background.js

```diff
@@ -246,6 +251,8 @@ passwordalert.background.setManagedPolicyValuesIntoConfigurableVariables_ =
       console.log('Managed policy found.  Enterprise use.');
       passwordalert.background.corp_email_domain_ =
           managedPolicy['corp_email_domain'].replace(/@/g, '').toLowerCase();
+      passwordalert.background.displayUserAlert_ =
+          managedPolicy['display_user_alert'];
```

Comment:
Down in passwordalert.background.handleManagedPolicyChanges_ the case statement also needs to handle the new display_user_alert